python/mlcroissant/pyproject.toml

```diff
@@ -66,3 +66,8 @@ exclude = ["recipes*"]
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+markers = [
+    "nonhermetic: Non-hermetic test cases (data from the internet)."
```

Comment:
Note: to launch hermetic tests, we have to launch `pytest -m "not nonhermetic"`. I don't like the double negation, but I don't see a better name. WDYT?